client/state/sites/plans/test/connections.js

```diff
@@ -0,0 +1,48 @@
+
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { spy } from 'sinon';
+/**
+ * Internal dependencies
+ */
+import { SITE_PLANS_FETCH_COMPLETED, SITE_PLANS_FETCH_FAILED } from 'state/action-types';
+import { connectFetchSitePlans } from '../connections';
+import { createWpcomSpy } from 'test/helpers/use-sinon';
+
+describe( 'connections', () => {
+	describe( '#connectFetchSitePlans()', () => {
+		const dispatchMock = spy();
+
+		it( 'should pass siteId to API call', () => {
+			const wpcom = createWpcomSpy(
```

Comment:
With https://github.com/Automattic/wp-calypso/pull/5935/files#r66920808 change in place that test would look like:

```js
const getSitePlans = spy();
connectFetchSitePlans( getSitePlans )( { siteId: 5 }, dispatchMock );
expect( getSitePlans.calledWith( 5 ) ).ok;
```